pages/dashboard/notifications.vue

```diff
@@ -60,12 +60,16 @@ export default {
     UpToDate,
   },
   async asyncData(data) {
-    const notifications = (
+    let notifications = (
       await axios.get(
         `https://api.modrinth.com/api/v1/user/${data.$auth.user.id}/notifications`,
         data.$auth.headers
       )
     ).data
+    notifications = notifications.sort(
```

Comment:
Same here, and remove the console.log, it will be causing errors in the future.

Reply:
Good catch, I really need to watch out for leaving console.log()s in my commits 😬